data/World/Water Temple MQ.json

```diff
@@ -2,6 +2,22 @@
     {
         "region_name": "Water Temple Lobby",
         "dungeon": "Water Temple",
+
+        "locations": {
+            "Water Temple MQ L1 Torch Pot 1": "at('Water Temple Dive', 'True') and (can_use(Iron_Boots) and (can_use(Hookshot) or can_play(Zeldas_Lullaby)))",
+            "Water Temple MQ L1 Torch Pot 2": "at('Water Temple Dive', 'True') and (can_use(Iron_Boots) and (can_use(Hookshot) or can_play(Zeldas_Lullaby)))",
+            "Water Temple MQ Lizalfos Hallway Pot 1": "at('Water Temple Dive', 'True') and (can_use(Iron_Boots) and (can_use(Hookshot) or can_play(Zeldas_Lullaby)))",
+            "Water Temple MQ Lizalfos Hallway Pot 2": "at('Water Temple Dive', 'True') and (can_use(Iron_Boots) and (can_use(Hookshot) or can_play(Zeldas_Lullaby)))",
+            "Water Temple MQ Lizalfos Hallway Pot 3": "at('Water Temple Dive', 'True') and (can_use(Iron_Boots) and (can_use(Hookshot) or can_play(Zeldas_Lullaby)))",
+            "Water Temple MQ Lizalfos Hallway Room Crate 1": "at('Water Temple Dive', 'True') and (can_use(Iron_Boots) or can_play(Zeldas_Lullaby))",
+            "Water Temple MQ Lizalfos Hallway Room Crate 2": "at('Water Temple Dive', 'True') and (can_use(Iron_Boots) or can_play(Zeldas_Lullaby))",
+            "Water Temple MQ Lizalfos Hallway Room Crate 3": "at('Water Temple Dive', 'True') and (can_use(Iron_Boots) or can_play(Zeldas_Lullaby))",
+            "Water Temple MQ Lizalfos Hallway Room Crate 4": "at('Water Temple Dive', 'True') and (can_use(Iron_Boots) or can_play(Zeldas_Lullaby))",
+            "Water Temple MQ Lizalfos Hallway Room Crate 5": "at('Water Temple Dive', 'True') and (can_use(Iron_Boots) or can_play(Zeldas_Lullaby))",
+            "Water Temple MQ Lizalfos Hallway Hall Crate 1": "at('Water Temple Dive', 'True') and (can_use(Iron_Boots) or can_play(Zeldas_Lullaby))",
+            "Water Temple MQ Lizalfos Hallway Hall Crate 2": "at('Water Temple Dive', 'True') and (can_use(Iron_Boots) or can_play(Zeldas_Lullaby))",
+            "Water Temple MQ Lizalfos Hallway Hall Crate 3": "at('Water Temple Dive', 'True') and (can_use(Iron_Boots) or can_play(Zeldas_Lullaby))"
```

Comment:
All of these checks should just be inside the Water Temple Dive region. Entering that regions requires Iron Boots, so you don't need to check again for Iron Boots on any of these. So the pots would all be hookshot or lullaby and the crates would all be True.

Trick: I suspect but can't be bothered to confirm, that basically all of the pots here could be hit with Bombchus. Should be a fun time for future me going around and seeing what all is actually possible, lol.

Trick: There is one crate here, the one closest to the upper exit of the Lizalfos hallway, that can actually be obtained without Irons, using Gold Scale and Bombchus. It even works as child, but it's a lot harder due to child being too short to clip under the raised gate, so the the dive needs to hit the switch perfectly on the edge. Eventually if that trick gets added, then that crate would have to be moved back out of the dive region and into this one. I will need to know which crate this one is -- I would guess lizalfos hallway crate 3. Is it 3?